avaframe/logging.conf

```diff
@@ -0,0 +1,32 @@
+[loggers]
+keys=root
+
+[handlers]
+keys=consoleHandler, fileHandler
+
+[formatters]
+keys=simple, complex
+
+[logger_root]
+level=INFO
+handlers=consoleHandler, fileHandler
+
+[formatter_simple]
+format=%(module)s - %(levelname)s - %(name)s -  %(message)s
+
+[formatter_complex]
+format=%(asctime)s - %(module)s : %(lineno)d- %(levelname)s - %(name)s  - %(message)s
+
+[handler_consoleHandler]
+class=StreamHandler
+formatter=simple
+level=INFO
+args=(sys.stdout,)
+
+[handler_fileHandler]
+class=handlers.TimedRotatingFileHandler
+interval=midnight
+backupCount=5
+formatter=simple
+level=DEBUG
+args=('%(logfilename)s',)
```

Comment:
This is a setup suitable for a constantly running service. 
I suggest we use a simple overwrite on the _logfilename_ file